### PR TITLE
style: reorder imports via prettier

### PR DESCRIPTION
**Pull request by schultzp2020**
## Description

Please explain the changes you made here.

Reorders the imports to maintain parity with the janus plugins repo. https://github.com/janus-idp/backstage-plugins/pull/419

## Which issue(s) does this PR fix

N/A

## PR acceptance criteria

Please make sure that the following steps are complete:

- GitHub Actions are completed and successful
- Unit Tests are updated and passing
- E2E Tests are updated and passing
- Documentation is updated if necessary
- Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: packages/app/src/App.tsx [hunk 1]
```diff
@@ -76,27 +66,18 @@ const routes = (
...
         <ReportIssue />
       </TechDocsAddons>
     </Route>
     <Route path="/create" element={<ScaffolderPage />} />
     <Route path="/api-docs" element={<ApiExplorerPage />} />
-    <Route
-      path="/tech-radar"
-      element={<TechRadarPage width={1500} height={800} id="default" />}
-    />
+    <Route path="/tech-radar" element={<TechRadarPage width={1500} height={800} id="default" />} />
```
Comment (PatAKnight) [inline packages/app/src/App.tsx:80]: So I see that a lot of these have been collapsed, I don't really have a preference either way but I wonder what other people think about the difference between the two? @christophe-f

**Review by PatAKnight — APPROVED** [review]
LGTM

**Review by christophe-f — CHANGES_REQUESTED** [review]
As discussed several times, we need to stay in parity with upstream. This PR will not be merged.

**Comment by schultzp2020** [comment]
> As discussed several times, we need to stay in parity with upstream. This PR will not be merged.

@christophe-f We have already merged a PR like this in https://github.com/janus-idp/backstage-plugins/pull/419. The only changes this PR is to the import order and does not diverge code logic wise from the upstream.

**Pull request closed without merging**